### ISSUE TITLE
Document mac mini ESXi setup

### ISSUE THREAD

**Issue by reaperhulk** [pyca/infra#49]
Credentials are dependent on #30 but write a bit about temporarily enabling ssh, shutting down VMs, running commands from https://esxi-patches.v-front.de/ESXi-6.5.0.html, rebooting, and booting the VMs back up.

Should also try to capture how to create new VMs I guess.

**Comment by reaperhulk** [comment]
ESXi 6.5 supports web UI administration via `https://<ip>/ui`. We should document that too.

**Comment by reaperhulk** [comment]
Also document how to create a mac builder. Steps are roughly:

* Make a new VM in Fusion
* Set up jenkins user (as an admin)
* Set up homebrew (and chown all the homebrew dirs so that group can write, this is for the jenkins update openssl job)
* Install pyenv and install 2.7.x
* Install 2.7, 3.4, 3.5, 3.6 from python.org (framework only)

**Comment by alex** [comment]
Closing as wontfix, we're planning on killing our macminis
